ocrd_tesserocr/crop.py

```diff
@@ -146,7 +151,19 @@ def process(self):
                     LOG.debug("Padded page border: %i:%i,%i:%i", min_x, max_x, min_y, max_y)
                     border = BorderType(Coords=CoordsType(
                         points_from_bbox(min_x, min_y, max_x, max_y)))
+                    # update PAGE (annotate border):
                     page.set_Border(border)
+                    # update METS (add the image file):
+                    page_image = page_image.crop(
+                        box=(min_x, min_y, max_x, max_y))
+                    file_id = input_file.ID.replace(self.input_file_grp, FILEGRP_IMG)
```

Comment:
Could this cause duplicate ID for METS not following our conventions for identifiers?

Reply:
Absolutely true! I forgot to capture this here (as I did with the PAGE result `file_id` at the end of the processor).

Reply:
done